src/library/index.js

```diff
@@ -0,0 +1,4 @@
+export * from './detail';
+export * from './form';
+export * from './header';
+export * from './list';
```

Comment:
Consider renaming these folders (including the `src/library` folder itself) to be more explicit about what kind of functionality they hold. To those of use working on the project, it's clear what these all refer to; however, to a future developer who is working on their first ticket in this repo, it might not be so obvious.

For example:
* `library` -> `libraries` or `library-authoring`
* `form` -> `create` or `create-library`
* etc.

Reply:
Good idea.  Done.